test/test_pickle.py

```diff
@@ -13,6 +13,32 @@
 
 import numpyro
 import numpyro.distributions as dist
+from numpyro.distributions.constraints import (
+    boolean,
+    circular,
+    corr_cholesky,
+    corr_matrix,
+    dependent,
```

Comment:
Do you want to test for `dependent`? It seems that this is causing lint failing. You can detect lint issues with `make lint` (see [contributing docs](https://github.com/pyro-ppl/numpyro/blob/master/CONTRIBUTING.md#development))

Reply:
my bad, missed that.